src/reducers/filterSets.js

```diff
@@ -1,253 +1,76 @@
-          color: '#EB6915'
-        },
-        {
-          slug: 'color-ultramarine-blue',
-          name: 'ultramarine-blue',
-          method: 'query',
-          type: 'match',
-          field: '_all',
-          term: 'ultramarine blue',
-          color: '#D8440E'
-        },
-        {
-          slug: 'color-ultramarine-blue',
-          name: 'ultramarine-blue',
-          method: 'query',
-          type: 'match',
-          field: '_all',
-          term: 'ultramarine blue',
-          color: '#CD130E'
-        },
-        {
-          slug: 'color-ultramarine-blue',
-          name: 'ultramarine-blue',
-          method: 'query',
-          type: 'match',
-          field: '_all',
-          term: 'ultramarine blue',
-          color: '#9D3469'
-        },
-        {
-          slug: 'color-ultramarine-blue',
-          name: 'ultramarine-blue',
-          method: 'query',
-          type: 'match',
-          field: '_all',
-          term: 'ultramarine blue',
-          color: '#6B2056'
-        },
-        {
-          slug: 'color-ultramarine-blue',
-          name: 'ultramarine-blue',
-          method: 'query',
-          type: 'match',
-          field: '_all',
-          term: 'ultramarine blue',
-          color: '#541439'
-        },
-        {
-          slug: 'color-ultramarine-blue',
-          name: 'ultramarine-blue',
-          method: 'query',
-          type: 'match',
-          field: '_all',
-          term: 'ultramarine blue',
-          color: '#6E6E6E'
-        },
-        {
-          slug: 'color-ultramarine-blue',
-          name: 'ultramarine-blue',
-          method: 'query',
-          type: 'match',
-          field: '_all',
-          term: 'ultramarine blue',
-          color: '#DCDCDC'
-        },
-        {
-          slug: 'color-ultramarine-blue',
-          name: 'ultramarine-blue',
-          method: 'query',
-          type: 'match',
-          field: '_all',
-          term: 'ultramarine blue',
-          color: '#F0F0F0'
-        },
-      ],
-    },
-    lines: {
-      title: "Lines",
-      slug: "lines",
-      options: {
-        composition: [
-          {
-            slug: 'line-vertical',
-            name: 'vertical'
-          },
-          {
-            slug: 'line-diagonal',
-            name: 'diagonal'
-          },
-          {
-            slug: 'line-horizontal',
-            name: 'horizontal'
-          },
-          {
-            slug: 'line-curvy',
-            name: 'curvy'
-          }
-        ],
-        linearity: [
-          {
-            slug: 'line-broken',
-            name: 'broken'
-          },
-          {
-            slug: 'line-unbroken',
-            name: 'unbroken'
-          },
-          {
-            slug: 'line-all-types',
-            name: 'all types'
-          }
-        ]
+const buildInitialState = () => {
+  var initialState = {
+    visibleFilterSet: null,
+    sets: {
+      colors: {
+        title: "Colors",
```

Comment:
Consider using single quotes instead of double quotes as a default for js/jsx files.
(https://github.com/airbnb/javascript#strings)